auth/CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+# CHANGELOG
+
+## Version 2.0.2
+
+New Features:
+
+1. Import the GO SDK with version 2.0.2.
```

Comment:
suggest to add the url of user guide: https://support.huaweicloud.com/devg-apisign/api-sign-sdk-go.html

Reply:
done.